### PR TITLE
feat: setup CI for envless

### LINKED ISSUE #331: [FEATURE]: Add CI setup for Build and Test for envless
### Description

CI setup for build and test for envless is being lacked as we need during out PR process on our github action workflow.

### Possible solution

Setup github action workflow having jobs with specific tasks.
- Install dependencies from package.json
- Build the project
- Test the project

### Alternatives

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Aju100**
# Description

Fixes #331 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I documented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Aju100** [comment]
cc @dahal  @chetannn

**Comment by Aju100** [comment]
BTW we can remove Vtest jobs as it is already integrated along with the build jobs. 

Future Changes:
- https://github.com/envless/envless/blob/main/.github/workflows/vitest.yml

**Comment by Aju100** [comment]
@chetannn can u inspect on this ?

**Comment by Aju100** [comment]
thank u @chetannn

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        - name: Generate Prisma Types
+          run: yarn run db:generate
+ 
+        - name: Build
+          run: yarn build
+          env:
+            SKIP_ENV_VALIDATION: true
+            EMAIL_FROM: envless@example.com
+ 
+        - name: Test
```
Comment (dahal) [inline .github/workflows/ci.yml:43]:
```suggestion
        - name: Unit Tests
          run: yarn test

        - name: E2E Tests
          run: yarn test:e2e
```

Lets make these changes and delete both of these actions

* https://github.com/envless/envless/blob/main/.github/workflows/playwright.yml
* https://github.com/envless/envless/blob/main/.github/workflows/vitest.yml

**Review by dahal — CHANGES_REQUESTED** [review]
Thanks for this amazing work @Aju100, I have made few recommendations and its good to go after 🚀

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Aju100) [reply .github/workflows/ci.yml:43]: what about linting one ? @dahal I missed this job does it have to be written ?

**Review by dahal — APPROVED** [review]

**Pull request merged**